src/dungeon/info.rs

```diff
@@ -0,0 +1,378 @@
+//! Contains information about dungeons that are not avaliable from the GW2 API.
+//! This information would have to be updated if dungeon rewards are reworked
+//! or if more dungeons are added.
+//!
+//! TODO: The amount of gold rewards are known for some story paths.
+//!       The missing story paths are currently using 0.
+
+/// Information about a dungeon.
+#[derive(Debug, PartialEq, Eq)]
+pub struct DungeonInfo {
+    /// The id used by the /v2/dungeons endpoint.
+    pub id: &'static str,
+    /// A user friendly short name.
+    pub short_name: &'static str,
+    /// A user friendly long name.
+    pub long_name: &'static str,
+    /// id of the achievement for collection all skins.
+    pub collection_id: u32,
+    /// id of the token used by the /v2/account/currencies endpoint.
+    pub currency_id: u32,
+    /// information about every path in this dungeon.
+    pub paths: &'static [PathInfo],
+}
+
+/// Information about a path within a dungeon.
+#[derive(Debug, PartialEq, Eq)]
+pub struct PathInfo {
+    /// The id used by the /v2/account/dungeons endpoint to indicate
+    /// if a user has done this path today.
+    pub id: &'static str,
+    /// A user friendly short name.
+    pub short_name: &'static str,
+    /// A longer user friendly name.
+    pub long_name: &'static str,
+    /// The index inserted into the bits array for the dungeon
+    /// frequenter achievement when this path is done.
+    pub dungeon_frequenter_index: Option<u8>,
+    /// The rewards for doing this path
+    pub rewards: Rewards,
+}
+
+/// Describes the rewards for a path.
+#[derive(Debug, PartialEq, Eq)]
+pub enum Rewards {
+    /// Story mission.  No token rewards.  Fixed coin reward.
+    Story { coins: u32 },
+    /// Explorable path.  100 tokens on first per day, 20 on repeat.
+    /// bonus_coins + 26s on first per day.  26s on repeat.
+    Explorable { bonus_coins: u32 },
```

Comment:
Is the intention here to hard code the base token reward?

Reply:
Currently, yes.  This is a internal implementation detail of the dungeon::Path accessors.  The layout of the PathInfo can be changed at any time and be non-breaking.

Reply:
Ah i see.